cluster/gce/util.sh

```diff
@@ -3578,11 +3578,11 @@ function kube-down() {
 
   # In the replicated scenario, if there's only a single master left, we should also delete load balancer in front of it.
   if [[ "${REMAINING_MASTER_COUNT}" -eq 1 ]]; then
+    local REMAINING_REPLICA_NAME="$(get-all-replica-names)"
+    local REMAINING_REPLICA_ZONE=$(gcloud compute instances list "${REMAINING_REPLICA_NAME}" \
+      --project "${PROJECT}" --format="value(zone)")
     if gcloud compute forwarding-rules describe "${MASTER_NAME}" --region "${REGION}" --project "${PROJECT}" &>/dev/null; then
       detect-master
```

Comment:
Don't you need to move this as well?

I.e. does detect-master set any env variable that is (can be) used by get-all-replica-names or the next line? AFAIK It sets PROJECT but maybe it's set here already.

Reply:
Added, to make sure that all variables are set.